object_database/web/content/components/Expands.js

```diff
@@ -2,6 +2,15 @@
  * Expands Cell Component
  */
 
+/** TODO/NOTE: It appears that the open/closed
+    State for this component could simply be passed
+    with the Cell data, along with what to display
+    in either case. This would be how it is normally
+    done in large web applications.
+    Consider refactoring both here and on the Cells
+    side
+**/
```

Comment:
given what we're doing, we should probably make a richer set of cells (e.g. a TreeView cell) instead of trying to cobble them together with individual pieces on the python side.